assets/js/stripe-payment-request.js

```diff
@@ -520,7 +535,7 @@ jQuery( function( $ ) {
 				});
 			} );
 
-			$( '.quantity' ).on( 'keyup', '.qty', function () {
+			$( '.quantity' ).on( 'keyup mouseup', '.qty', wc_stripe_payment_request.debounce( 250, function() {
```

Comment:
In the context of `mouseup` and `keyup`, I think you can simply use the `change` event. It works immediately both in Chrome and Safari.

Reply:
I'm not going be able to test these again before the next sprint so leaving it as is.

Reply:
Sorry again, had a moment to do a quick test and this works nicely. Fixed up in: 
e41344d